make/cluster.sh

```diff
@@ -94,6 +93,8 @@ while [ $# -ne 0 ]; do
   shift
 done
 
+kind_cluster_name=${name-kind}
```

Comment:
where is `name-kind` defined?

Reply:
This is using [bash parameter expansion](https://www.gnu.org/software/bash/manual/bash.html#Shell-Parameter-Expansion). I just modified it to use `:-` instead of `-`. According to the doc: 

> ${parameter:-word}
> If parameter is unset or null, the expansion of word is substituted. Otherwise, the value of parameter is substituted.

In [this block of code](https://github.com/malovme/cert-manager/blob/master/make/cluster.sh#L71-L81), we set the `name` variable to its value based on the command line arguments. However, before this commit, we never used that value anywhere. 

Believe me, I had to stare at this for a few minutes to know what was going on. 

Thanks for taking a look!

Reply:
Thank you for figuring this out, could you move the default value to the top of the file (like before) and just add the `kind_cluster_name=${name}` line to fix the bug?

NOTE to self: this file needs some work (removing the TODOs).

Reply:
> could you move the default value to the top of the file (like before) 
No, because the variable `name` is not defined until after the `case` statment where the arguments are parsed. 

> It would be amazing if you could add a comment to those 3 variables
Will do.